lte/gateway/fabfile.py

```diff
@@ -688,16 +683,6 @@ def build_and_start_magma(gateway_host=None, destroy_vm='False', provision_vm='F
     sudo('service magma@magmad start')
 
 
-def make_integ_tests(test_host=None, destroy_vm='False', provision_vm='False'):
-    destroy_vm = strtobool(destroy_vm)
-    provision_vm = strtobool(provision_vm)
-    if not test_host:
-        vagrant_setup('magma_test', destroy_vm, force_provision=provision_vm)
-    else:
-        ansible_setup(test_host, "test", "magma_test.yml")
-    execute(_make_integ_tests)
-
-
```

Comment:
I also thought this would be superfluous. But that is still used at the following place:
https://github.com/magma/magma/blob/ff3fa473a60333cf91c18bba71e5eab078be8653/lte/gateway/python/integ_tests/federated_tests/fabfile.py#L268
I have probably overlooked it, but you did covered for that, right?

Reply:
See changes in `lte/gateway/python/integ_tests/federated_tests/fabfile.py` - the call is removed.